components/Website_Layout/Navbar.js

```diff
@@ -26,6 +41,17 @@ export default function navbar() {
       <Link href="/MerchandisePage">
         <a style={tabStyle}>Buy our Merchandise!</a>
       </Link>
+      {loggedIn ? <p>{username}</p>:
```

Comment:
add a profile pic if the user is logged in

Reply:
done